tests/feedme/phase2/analytics/test_health_monitoring.py

```
@@ -0,0 +1,667 @@
+            mock_net.return_value.bytes_recv = 1024 * 1024 * 150  # 150MB
+            
+            # Act
+            resource_health = await health_monitor.check_system_resources()
+            
+            # Assert
+            assert resource_health['cpu_usage_percent'] == 85.5
+            assert resource_health['memory_usage_percent'] == 82.0
+            assert resource_health['disk_usage_percent'] == 75.0
+            
+            # Should trigger alerts for high CPU and memory usage
+            assert resource_health['alerts']['cpu_high'] == True
```

Ruff (0.11.9)

Avoid equality comparisons to `True`; use `if resource_health['alerts']['cpu_high']:` for truth checks

Replace with `resource_health['alerts']['cpu_high']`

(E712)